tcs/sco2_cycle_components.cpp

```diff
@@ -1529,7 +1529,7 @@ int C_comp_multi_stage::design_given_outlet_state(int comp_model_code, double T_
 
     double max_calc_tip_speed = mv_c_stages[0]->ms_des_solved.m_tip_ratio;      //[-]
 
-	double tip_speed_limit = 0.85;
+    double tip_speed_limit = 0.8;
```

Comment:
What's the reason for this change?

Reply:
The more restrictive value (i.e., 0.85) was causing the simple cycle off-design calculations to fail due to the stage design breaking this limit. This change got the code working; however, I didn't dig into the implications. I could send you a small example of the code breaking case if you would like.

Reply:
I couldn't find the case that was causing this issue. I have reverted this change.